docs/en/integrations/data-visualization/tableau/tableau-and-clickhouse.md

```diff
@@ -0,0 +1,172 @@
+    - macOS: `~/Library/Tableau/Drivers`
+    - Windows: `C:\Program Files\Tableau\Drivers`
+5. Configure a ClickHouse data source in Tableau and start building data visualizations!
+
+## Configure a ClickHouse data source in Tableau
+
+Now that you have the `clickhouse-jdbc` driver installed and set, let's see how to define a data
+source in Tableau that connects to the **TPCD** database in ClickHouse.
+
+1. Start Tableau. (If you already had it running, then restart it.)
+
+2. From the left-side menu, click on **More** under the **To a Server** section. Search for **ClickHouse by ClickHouse** in the available connectors list:
+
+   ![ClickHouse JDBC](../images/tableau_connecttoserver.png)
+<br/>
+
+:::note
+Don't see the **ClickHouse by ClickHouse** connector in your connectors list? It might be related to an old Tableau Desktop version.
+To solve that, consider upgrading your Tableau Desktop application, or [install the connector manually](#install-the-connector-manually).
+:::
+
+3. Click on **ClickHouse by ClickHouse**  and the following dialog will pop up:
+
+   ![ClickHouse JDBC Connector Details](../images/tableau_connector_details.png)
+ 
+4. Click **Install and Restart Tableau**. Restart the application.
+5. After restarting, the connector will have its full name: `ClickHouse JDBC by ClickHouse, Inc.`. When clicking it the following dialog will pop up:
+
+   ![ClickHouse JDBC Connector Details Details](../images/tableau_connector_dialog.png)  
+
+6. Enter your connection details:
+
+    | Setting  | Value                                                  |
+    | ----------- |--------------------------------------------------------|
+    | Server      | **Your ClickHouse host (with no prefixes or sufixes)** |
+    | Port   | **8443**                                               |
+    | Database | **default**                                            |
+    | Username | **default**                                            |
+    | Password | *\*****                                                |
+
+:::note
+When working with ClickHouse cloud, it's required to enable the SSL checkbox for secured connections.
+:::
+<br/>
+
+
+:::note
+Our ClickHouse database is named **TPCD**, but you must set the **Database** to **default** in the dialog above, then
+select **TPCD** for the **Schema** in the next step. (This is likely due to a bug in the connector, so this behavior
+could change, but for now you must use **default** as the database.)
+:::
+
+7. Click the **Sign In** button and you should see a new Tableau workbook:
+
+   !["New Workbook"](../images/tableau_newworkbook.png)
+
+8. Select **TPCD** from the **Schema** dropdown and you should see the list of tables in **TPCD**:
+
+   !["Select TPCD for the Schema"](../images/tableau_tpcdschema.png)
+
+You are now ready to build some visualizations in Tableau!
+
+## Building Visualizations in Tableau
```

Comment:
It's more about Tableau usage than ClickHouse. Not sure we need this in the docs (or on this page). cc @laeg

Reply:
I was there historically before this PR. Let me know if you want me to remove it.